dash/validator_conn_executor.go

```diff
@@ -163,58 +165,47 @@ func (vc *ValidatorConnExecutor) handleValidatorUpdateEvent(event types.EventDat
 		return nil // not really an error
 	}
 	vc.validatorSetMembers = newValidatorMap(event.ValidatorUpdates)
+	if len(event.QuorumHash) > 0 {
+		if err := vc.setQuorumHash(event.QuorumHash); err != nil {
+			vc.BaseService.Logger.Error("received invalid quorum hash", "error", err)
```

Comment:
since `BaseService` is embedded structure, then to reach out the `Logger` you can use this expression `vc.Logger.Error(...`